agent/bench-scripts/tests/pbench-trafficgen/test-51.txt

```diff
@@ -0,0 +1,310 @@
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/2-default/sample3
+Iteration 2-profile1-0.001pct_drop complete (2 of 4), with 1 pass and 0 failures
+cp: cannot stat 'profile2': No such file or directory
+Starting iteration[3-profile2-0.002pct_drop] (3 of 4)
+test sample 1 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample1
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample1
+test sample 2 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample2
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample2
+test sample 3 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample3
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/3-default/sample3
+Iteration 3-profile2-0.002pct_drop complete (3 of 4), with 1 pass and 0 failures
+cp: cannot stat 'profile2': No such file or directory
```

Comment:
Seems like copy operations are failing

Reply:
Yes, the profile files were missing; I have added mock profile files to the test.